### PR TITLE
setup_host: Changes to get GVT-d working

### PR DESCRIPTION
**Pull request by JeevakaPrabu**
- Set i915.force_probe to * to force probe the driver for
new Intel graphics devices that are recognized but not properly
supported by this kernel version.
- Update mesa package that provides GLX and DRI capabilities
- Put system in text mode

Tracked-On: OAM-90861
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>

### CODE REVIEW

**Comment by JeevakaPrabu** [comment]
@renchenglei @feijiang1 @lyintel @YadongQi @Shao-Feng

**Comment by JeevakaPrabu** [comment]
@sgnanase @swaroopbalan

**Comment by JeevakaPrabu** [comment]
@kaushlen

**Comment by JeevakaPrabu** [comment]
@lchen43

File: groups/device-specific/caas/setup_host.sh [hunk 1]
```diff
@@ -118,12 +118,13 @@ function ubu_enable_host_gvtg(){
...
 function ubu_enable_host_gvtd(){
-	if [[ ! `cat /etc/default/grub` =~ "intel_iommu=on kvm-intel.nested=y" ]]; then
+	systemctl set-default multi-user.target
+	if [[ ! `cat /etc/default/grub` =~ "intel_iommu=on kvm-intel.nested=y i915.force=probe=* " ]]; then
 		read -p "The grub entry in '/etc/default/grub' will be updated for enabling GVT-d, do you want to continue? [Y/n]" res
 		if [ x$res = xn ]; then
 			exit 0
 		fi
-		sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"intel_iommu=on kvm-intel.nested=y /g" /etc/default/grub
+		sed -i "s/GRUB_CMDLINE_LINUX=\"/GRUB_CMDLINE_LINUX=\"intel_iommu=on kvm-intel.nested=y i915.force_probe=* /g" /etc/default/grub
```
Comment (Shao-Feng) [inline groups/device-specific/caas/setup_host.sh:127]: only need to set as "intel_iommu=on". force_probe is not needed for GVT-d

File: groups/device-specific/caas/setup_host.sh [hunk 2]
```diff
@@ -155,7 +156,7 @@ function check_kernel(){
 			for(i=0;i<NF;i++) { if ($i == "Linux" && $(i+1) == "version") { print $(i+2); next; } }
 		}'
 	)
-	if [[ "$version" > "5.0.0" ]]; then 
+	if [[ "$version" > "5.0.0" ]]; then
```
Comment (Shao-Feng) [inline groups/device-specific/caas/setup_host.sh:159]: For supporting new platform, we use 5.4.X now

File: groups/device-specific/caas/setup_host.sh (hunk 1 above)
Comment (Shao-Feng) [reply groups/device-specific/caas/setup_host.sh:127]: force_probe is only needed on EHL and Tigerlake. need to check the platform before setting this one. and use VGA PCI id to replace "*"

Comment (JeevakaPrabu) [reply groups/device-specific/caas/setup_host.sh:127]: Agreed but setting to force_probe is not causing any issues with KBL and CML. force_probe is to Force probe the driver for new Intel graphics devices that are recognized but not properly supported by this kernel version. Not a major issue

Comment (Shao-Feng) [reply groups/device-specific/caas/setup_host.sh:127]: If no error will be caused, I am OK to force_probe here in GVT-d case

File: groups/device-specific/caas/setup_host.sh (hunk 2 above)
Comment (JeevakaPrabu) [reply groups/device-specific/caas/setup_host.sh:159]: This will be taken care in new patch

**Pull request merged**